lib/ts/supertokens.ts

```diff
@@ -48,7 +48,16 @@ export default class SuperTokens {
 
     telemetryEnabled: boolean;
 
+    debugEnabled: boolean;
```

Comment:
this variable should not be required, and even if it is, it should be in the logger.ts file

Reply:
It was only used in the tests, removed it.